src/modules/package.ts

```diff
@@ -68,6 +69,10 @@ function getPackageFolderPath(): string {
     return join(getVelocitasRoot(), 'packages');
 }
 
+export function getPackageRepo(packageName: string): string {
```

Comment:
this is the same like: const PACKAGE_REPO = (packageName: string) => `${GITHUB_API_URL}${GITHUB_ORG_ENDPOINT}/${packageName}`;

and if you want to use it like that this probably does not need to be exported